bin/copy-labels.js

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs');
+
+const { cli } = require('../lib/cli.js');
```

Comment:
I'd prefer to use `yargs`.

Reply:
@belochub mentioned that we shouldn't add dependencies in this repository.

Reply:
I don't see the need in such complex CLI interface as in this implementation for the purpose of GitHub label management, I think it can be simplified to the point where there will be no need to implement your own CLI package (or some part of it) inside this package. For example, I don't think we need any shorthands for options, autogenerated help, argument types.